vaultvisor/src/vaultvisor.rs

```diff
@@ -0,0 +1,148 @@
+use crate::error::Error;
+use codec::{Decode, Encode};
+use jsonrpsee::{
+    core::{
+        client::{Client as WsClient, ClientT},
+        JsonValue,
+    },
+    rpc_params,
+    ws_client::WsClientBuilder,
+};
+use sp_core::Bytes;
```

Comment:
Should be able to use `subxt::sp_core` I think, if so please remove the dependency.

Reply:
There is no dependency on subxt atm afaik, so better to use sp_core directly in that case I think